app/components/course-stage-screencast-preview.hbs

```diff
@@ -20,7 +20,7 @@
     <div class="flex gap-3 mt-3">
       <div class="flex items-center">
         {{svg-jar "clock" class="w-4 text-gray-300 mr-1.5"}}
-        <div class="text-gray-600 text-xs">{{round (div @screencast.durationInSeconds 60)}} minutes</div>
+        <div class="text-gray-600 text-xs" data-test-screencasts-formatted-duration>{{format-time-duration @screencast.durationInSeconds}}</div>
```

Comment:
@ryan-gang let's convert this component to typescript & remove the `@glint-nocheck` (you can replace it with `{{@glint-expect-error}}` if any sub-components aren't type-safe. 

As a practice, let's always convert components when we make changes to them. 

In this case for example, the newly added helper needs to be added to the Glint registry - something that we didn't catch because this component doesn't have typechecking